package.json

```diff
@@ -52,12 +52,12 @@
     "unfetch": "^4.0.1"
   },
   "devDependencies": {
-    "@storybook/addon-actions": "^3.4.7",
-    "@storybook/addon-knobs": "^3.4.7",
-    "@storybook/addon-links": "^3.4.7",
-    "@storybook/addon-options": "^3.4.7",
-    "@storybook/addons": "^4.0.0-alpha.24",
-    "@storybook/vue": "^4.0.0-alpha.24",
+    "@storybook/addon-actions": "^4.1.7",
+    "@storybook/addon-knobs": "^4.1.7",
+    "@storybook/addon-links": "^4.1.7",
+    "@storybook/addon-options": "^4.1.7",
+    "@storybook/addons": "^4.1.7",
+    "@storybook/vue": "^4.1.7",
```

Comment:
did you test storybook with these major version updates?

Reply:
No, I haven't. I've reverted the versions.